src/main/java/io/jenkins/plugins/conventionalcommits/utils/GradleProjectType.java

```diff
@@ -40,4 +40,8 @@ public Version getCurrentVersion(File directory, ProcessHelper processHelper)
     }
     return Version.valueOf(version);
   }
+
+  @Override
+  public void writeVersion(File directory, Version nextVersion, ProcessHelper processHelper)
+      throws IOException, InterruptedException {}
```

Comment:
for the not yet implemented methods, we could throw a NotImplementedException?

Reply:
Yes, I will add that! Thank you!

PS:
The reason I did not do that:
1. It would get added in writeVersion as throws 
2. That would change the abstract class declaration as well
3. MavenProjectType's `writeVersion` would also have the NotImplementedException in its throws.. 😅